account/notifications.md

```diff
@@ -47,12 +48,12 @@ result events on an endpoint of your application without having to listen to
 all events via the Websocket API. No more lost events and having to reconnect
 to the websocket if and when it fails.
 
-You will notice a **Webhooks** button at the top of the Notifications window,
-there you will find an *Add* button which will allow you to create a new
-Webhook.
+To create a webhook stay on the **Notifications** panel:
+1) click on **Webhooks** button at the top of the Notifications window.
```

Comment:
Same for the list here. You are also missing a uppercase at the start of the sentence